10_Scripts/DEADataHandling.py

```diff
@@ -451,7 +451,8 @@ def load_clearlandsat(dc, query, sensors=('ls5', 'ls7', 'ls8'), product='nbart',
         # Optionally filter to replace no data values with nans
         if mask_invalid_data:
 
-            print('    Replacing invalid -999 values with NaN (data will be coerced to float64)')
+            print('    Replacing invalid -999 values with NaN (data will be coerced to float32)')
```

Comment:
alignment is off?

Reply:
How do you tell this? It seems to look OK to me...

Reply:
maybe it's just github diff, I'll checkout code locally

Reply:
@robbibt there are tab characters on `combined_ds` line, we should really use spaces everywhere in python

Reply:
Ah, weird, I thought I had all my editors set up to avoid that. Will have a look and replace them with spaces

Reply:
Fixed the tab issue